Formula/cockroach-sql.rb

```diff
@@ -0,0 +1,29 @@
+# Auto-generated file, DO NOT EDIT
+# Source: release/cockroach-sql-tmpl.rb
+
+class CockroachSql < Formula
+  desc "Distributed SQL database shell"
+  homepage "https://www.cockroachlabs.com"
+  version "22.2.0-beta.5"
```

Comment:
Do we want pre-releases? (For other brew formulas, we currently only do this for latest stable releases) 

If so, maybe this initial formula is for v22.1.{latest-patch}.

Reply:
Sorry, I wasn't clear regarding this. The idea is to merge this after #117, which implies 22.2.0 released. At that point I would need to update the PR and regenerate this file with 22.2.0.

Reply:
Oh, maybe [comment](https://github.com/cockroachdb/homebrew-tap/pull/117#issuecomment-1300729435) from applies here as well?

> Do not merge until 22.2.0 is stable